substrate/frame/system/src/lib.rs

```diff
@@ -2286,6 +2286,7 @@ pub fn unique(entropy: impl Encode) -> [u8; 32] {
 }
 
 /// Event handler which registers a provider when created.
+#[deprecated(note = "Use `inc_providers` & `dec_providers` methods directly instead.")]
```

Comment:
No removal date here?

Reply:
Same 6months, from the PR start date?

Reply:
Yea, I think i linked this doc in the original issue https://github.com/paritytech/polkadot-sdk/blob/master/docs/contributor/DEPRECATION_CHECKLIST.md